man/systemd-dissect.xml

```diff
@@ -386,6 +386,25 @@
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--loop-ref=</option></term>
+
+        <listitem><para>Configures the "reference" string the kernel shall report as backing file for the
+        loopback block device. While this is supposed to be a path or filename referencing the backing file
```

Comment:
Not sure it is correct English grammar, but I'd like to add `,` after `the backing file`.